src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+
+	public static final String EXAMPLE = "123";
+	public static final String MESSAGE_BLOCK_CONSTRAINTS = "Block numbers should only contain numbers and letters.";
+	//	    public static final String BLOCK_VALIDATION_REGEX = "\\d+";
+	/**
+	 * @return the example
+	 */
+	public static String getExample() {
```

Comment:
Better use method description for trivial method, like `Returns ...`. BTW, the method name is not clear.